### ISSUE TITLE
A confusing icon on "Activity Diary" 's UI

### ISSUE THREAD

**Issue by b450mhdv** [ramack/ActivityDiary#302]
@ramack 

Hello, I found a problem with the app's UI design. When I created a new activity, I thought the widget in the upper right corner would be used to clear my inputs. But the "trash" button can delete the total activity and return to the homepage. The common "trash" icon I saw in other popular apps may usually delete all items on one page but do not return to the previous page. In short, I think the "delete" button is necessary on the "modify" page, but may not be needed on the "New Activity'' page because users can directly delete it by clicking the "X" button. So, just as a suggestion, if you can redesign it, it may greatly improve the user experience. Could you try to fix this issue?

![ActivityDiary (1)](https://user-images.githubusercontent.com/78015408/128504529-ff3db064-ffea-4bca-a11a-222a5557f7c7.png) ![ActivityDiary (2)](https://user-images.githubusercontent.com/78015408/128504533-0adcd1fc-3875-4958-9774-4333b408bc90.png)

**Comment by ramack** [comment]
Yes you're totally right, this should be hidden when creating a new activity. Thanks for the suggestion.
